README.md

```diff
@@ -17,61 +17,64 @@ This [Snap](https://snapcraft.io/) is part the
 
 - 64bit-X86 machine
 - Baremetal strongly recommended
-- Two ethernet interfaces (SGi and S1)
+  - AMD64 dual-core processor around 2GHz clock speed or faster
```

Comment:
As mentioned in the other comment below, I've removed all the deployment details from README. Now it simply points user to how-to guide. 
I left system requirements in how-to, because I see it as sort of prerequisite for it.